test/reports/bootstrap-voting/rest-gateway-0-rest.json

```diff
@@ -88,4 +91,4 @@
     "deploymentToolVersion": "abc",
     "lastUpdatedDate": "2021-05-23"
   }
-}
+}
```

Comment:
[multiple files]
please make sure files end with a newline. you(r editor) seems to have truncated this in most of the json files.